src/Repository/LineItemRepository.php

```diff
@@ -61,18 +62,18 @@ public function findAllAsCollection(): LineItemCollection
     /**
      * @throws EntityNotFoundException
      */
-    public function findOneById(int $id): LineItem
+    public function findOneById(UuidV6 $id): LineItem
     {
         $lineItem = $this->createQueryBuilder('l')
             ->select('l')
             ->where('l.id = :id')
-            ->setParameter('id', $id)
+            ->setParameter('id', $id, 'uuid')
```

Comment:
Can we have a way we are working with uuids in repos unified more or less?
Meaning that in line 70 we are using third arg to tell Doctrine the type,
but on line 93 we are converting to binary form
p.s.
I know that for line 93, and `... IN()...` it's easier to go with binary form, but why not using `toBinary` call in there?

Reply:
Checking

Reply:
Do you have any suggestion @uncleempty ?